submissions/korotushak96/js-dom/script.js

```diff
@@ -0,0 +1,94 @@
+            image: 'img/Bowtruckle.png'
+        },
+        {
+            id: 4,
+            name: 'Nundu',
+            description: 'The Nundu is a large East African beast that resembles a leopard. It moves silently, despite its "gigantic" size and is considered by some to be the most dangerous creature alive. The breath of the Nundu is toxic and filled with disease so potent that it can wipe out entire villages of people',
+            image: 'img/nundu_by_bear_drool_dbd2elu-pre.png'
+        },
+        {
+            id: 5,
+            name: 'Jarvey',
+            description: 'The Jarvey resembles an overgrown ferret, and is commonly found in Great Britain, Ireland, and North America. The Jarvey is capable of Human speech, although true conversation with a Jarvey is impossible. The creature uses short, usually rude, statements and phrases in an almost constant stream.',
+            image: 'img/Jarvey.png'
+        },
+        {
+            id: 6,
+            name: 'Murtlap',
+            description: 'The Murtlap is a marine beast resembling a rat with a growth on its back resembling a sea anemone, found on the coastal areas of Britain.',
+            image: 'img/Murtlap.jpg'
+        },
+    ]
+
+    let menuHTML = '';
+    let descriptionHTML = '';
+
+    beastsData.forEach((beast)=>{
+        const {id, name, description, image} = beast;
+        // create menu-items
+        menuHTML += `<div class='menu-item' data-id='${id}'>${name}</div>`;
+        // and description
+        descriptionHTML += 
+            `<div class='content-item'>
+                <div class='content-item_text'>${description}</div>
+                <img class='content-item_image' src='${image}'></img>
+            </div>`
+    });
+
+    menu.insertAdjacentHTML('afterbegin', menuHTML);
+    content.insertAdjacentHTML('afterbegin', descriptionHTML);
+
+    const contentItem = document.querySelectorAll('.content-item');
+    const menuItem = document.querySelectorAll('.menu-item');
+
+    burger.addEventListener('click', ()=>{
+        burger.classList.toggle('change');
+        menu.classList.toggle('open');
+    });
+
+    switchActive=(i)=>{
+        let chosen = document.querySelector('.chosen');
+        if (chosen) {
+            let activeId = chosen.dataset.id - 1;
+            menuItem[activeId].classList.remove('chosen');
+            contentItem[activeId].classList.remove('active')
+        }
+        // menuItem[idx].classList.remove('chosen');
+        contentItem[i].classList.toggle('active');
+        menuItem[i].classList.toggle('chosen');
+    };
+
+    switchActive(1);
+
+    menu.addEventListener('click', (event)=>{
+        let {target} = event;
+        menuItem.forEach((item, i)=>{
```

Comment:
Nice work! The last fix and we are ready for merging, I promise :)
Try to change `forEach` to `Array.prototype.find` method.
Let's imagine this case: user clicks to the first menu item, so we take the first beast and make it visible.
No need to iterate through all other menu items and to check menu item indexes again.

Reply:
Ok. I implemented it. Please, look at my it =)